spec/react_on_rails/utils_spec.rb

```diff
@@ -294,7 +294,7 @@ module ReactOnRails
               allow(Rails).to receive(:version).and_return("4.2")
             end
 
-            it "memoizes the result" do
+            it "memorizes the result" do
```

Comment:
memoize was correct.
```suggestion
            it "memoizes the result" do
```